### DISCUSSION TITLE
Save the daily count of pomodoros to a database and show it as heatmap

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [zidoro/pomatez#568] [Ideas]
As a person who works with pomodoro system for a while and also has his own type of pomodoro method, I usually record the count of pomodoros that I used for each task, and at the end of the day write them into my daily journal.
Pomatez is a very intuitive app, but whenever I restart my computer, the number of Pomodoros will be 0 again.
It will be awesome if we can have it any where saved, and also export the data in csv format for further analysis. (Some cool graphs inside the app will also be very cool feature).

**Comment by sekwah41** [comment]
I believe we've had an issue raised as a suggestion with a similar request. There are a few more things for me to focus on first though things like cloud saves and this would make for some nice enhancements.

Tbh this would be a nice local enhancement but I want to revamp how it stores a lot of the data that would be required for this especially with mobile in mind.
